cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -245,11 +249,12 @@ func TestEvaluateExecute(t *testing.T) {
 				filepath.Join("result-directory", "golang-summed.csv"): func(t *testing.T, filePath, data string) {
 					actualAssessments := validateMetrics(t, extractMetricsCSVMatch, data, []metrics.Assessments{
 						metrics.Assessments{
-							metrics.AssessmentKeyCoverage:         20,
-							metrics.AssessmentKeyFilesExecuted:    2,
-							metrics.AssessmentKeyResponseNoError:  2,
-							metrics.AssessmentKeyResponseNoExcess: 2,
-							metrics.AssessmentKeyResponseWithCode: 2,
+							metrics.AssessmentKeyCoverage:                      20,
+							metrics.AssessmentKeyFilesExecuted:                 2,
+							metrics.AssessmentKeyFilesExecutedMaximumReachable: 2,
```

Comment:
@bauersimon One thing I don't like is that this value is also summed , which I guess it doesn't make sense

Reply:
I think it does... it holds the maximum reachable files per repository it will work out once the summed csv are removed